lib/vector-vrl/web-playground/build.rs

```diff
@@ -18,6 +19,16 @@ fn get_vector_toml_path() -> PathBuf {
         .to_path_buf()
 }
 
+fn get_git_hash() -> String {
+    let output = Command::new("git")
+        .args(&["rev-parse", "HEAD"])
+        .output()
+        .unwrap();
```

Comment:
Nit: Using `expect` here and the other places you are using `unwrap` be nice to give more context on the failure. I realize this only happens at build time though and the error will be fairly easy to track down based on the stacktrace.